packages/grid/frontend/frontend.dockerfile.dockerignore

```diff
@@ -0,0 +1,10 @@
+# Frontend
```

Comment:
MAYBE: we could keep the file names short like
frontend.dockerignore?

Reply:
similar for backend and frontend

Reply:
That's how docker wants it - https://docs.docker.com/build/building/context/#filename-and-location

Or we move it under packages and combine all ignores into one - which might create conflicts in future. So better to keep it separate and explicit.

Reply:
Sure the naming makes sense 👍